function-reference/xml2json.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Convert xml string to json string
+---
+
+# xml2json
+This uses the [xml-js](https://github.com/nashwaan/xml-js) library
```

Comment:
library also seems incorrect since we've updated the library

Reply:
Looks like you reviewed an old version somehow. Check the latest changes